package.json

```diff
@@ -54,7 +54,7 @@
   ],
   "types": "./lib/index.d.ts",
   "dependencies": {
-    "@firebase/database": "^0.4.7",
+    "@firebase/database": "^0.4.8",
```

Comment:
You will have to run `npm install` on your working copy and push the updated package lock file for the tests to pass.

Reply:
Actually, the firebase-js-sdk won't be exposing the variable so we won't need to change this at all.